### ISSUE TITLE
Failed to import games from Indiegala. Object reference not set to an instance of an object.

### ISSUE THREAD

**Issue by grumpybunny** [Lacro59/playnite-indiegala-plugin#19]
Get the error in the title when attempting to import Indiegala. Playnite 9.4, extension v 2.0.1 

Same behaviour whether "Use client" is checked or not (these logs were captured with the box ticked). When viewing the extension settings, it hangs on "Checking authentication status" until I manually reauthenticate, but will revert to the endless checking behaviour on app restart. 

Fresh logs attached.

[extensions.log](https://github.com/Lacro59/playnite-indiegala-plugin/files/7357000/extensions.log)
[playnite.log](https://github.com/Lacro59/playnite-indiegala-plugin/files/7357001/playnite.log)

**Comment by Hernald** [comment]
So am I.

**Comment by Hernald** [comment]
The new version failed for me but this time left no error warning.

**Comment by grumpybunny** [comment]
We seem to be making some kind of progress - A game I picked up on IG today (Colortone) showed up when updating the library just now.

2 others (Exit Limbo: Opening and Renzo Racer) which were added sometime between the Playnite 9 update and now still aren't being picked up and I'm not sure why. Here are some new logs captured with the debug version of the add-on installed.

[extensions.log](https://github.com/Lacro59/playnite-indiegala-plugin/files/7378064/extensions.log)
[playnite.log](https://github.com/Lacro59/playnite-indiegala-plugin/files/7378065/playnite.log)
